app/assets/stylesheets/arclight/modules/sidebar.scss

```diff
@@ -1,3 +1,12 @@
 .al-in-person-repository-location {
   clear: left;
 }
+
+.al-digital-object {
+  margin-bottom: $spacer;
+}
+
+// Hiding this from display since we don't need it.
+dt.blacklight-digital_objects_ssm {
+  display: none;
```

Comment:
Wasn't quite sure what to do here.  This goes w/ Blacklight's default rendering and then hides the label that we don't want w/ `display: none`.  We could do a screen-reader only approach, but it wasn't obvious that this label was actually necessary.

We could also just do something completely custom, but then coordinating which accordion section should be open first might get a bit ugly.